cogment_verse/specs/environment_specs.py

```diff
@@ -86,16 +87,23 @@ def get_action_space(self, actor_class=PLAYER_ACTOR_CLASS, seed=None):
         return ActionSpace(deserialize_gym_space(self._pb.action_space), actor_class, seed)
 
     @classmethod
-    def create_homogeneous(cls, num_players, turn_based, observation_space, action_space):
+    def create_homogeneous(
+        cls,
+        num_players,
+        turn_based,
+        observation_space,
+        action_space,
+        serilization_format=SerializationFormat.STRUCTURED,
```

Comment:
Specs should always be structured.

Reply:
I set the same default value as these functions  `serialize_gym_space(...)` and `serialize_gym_space(...)` but users can override them according to the gym environment